styles.css

```diff
@@ -24,14 +37,12 @@ nav {
 	flex-direction: row;
 	flex-wrap: nowrap;
 	justify-content: space-around;
```

Comment:
perhaps space between would be better, do you want space on the edge of either side of your nav buttons? If so use margin & padding.